paymentsheet/src/main/java/com/stripe/android/paymentsheet/analytics/DefaultEventReporter.kt

```diff
@@ -51,6 +53,7 @@ internal class DefaultEventReporter @Inject internal constructor(
     }
 
     override fun onLoadSucceeded(
+        savedSelection: Deferred<SavedSelection>,
         linkEnabled: Boolean,
         currency: String?,
     ) {
```

Comment:
I made this deferred so that load times don't get an unexpected spike from Google Pay readiness loading times. We can stop loading in the reporter then wait for the saved selection to be ready.

Reply:
Do we actually want this? I think our analytics should reflect what the user is seeing. I don't think the user sees the screen unless google pay returns, right?

Reply:
Correct, I wasn't sure if avoiding the load time of google pay was something we did on purpose. I do agree it's better to track the loading time + google pay loading time. I'll update that here.